scripts: logging: dictionary: Add uart support

harish1992 opened the pull request:
This commit adds support to dictionary logging to parse binary data directly from input serial and generate the ascii logs.

nordic-krch commented on scripts/logging/dictionary/log_parser_uart.py:
```diff
@@ -0,0 +1,65 @@
+    else:
+        logger.setLevel(logging.INFO)
+
+    # Parse the 2Kbs of log every second from uart
```
2k/sec seems small. I guess that we can easily get more data than that. Not sure what is better, increase it or decrease the sleep time.

nordic-krch commented on scripts/logging/dictionary/log_parser_uart.py:
```diff
@@ -0,0 +1,65 @@
+                data = ser.read(2048)
+                parser.parser(data, args.dbfile, logger)
+            else:
+                print('no data')
```
i would remove that print.

harish1992 replied:
Will take care of this

nordic-krch reviewed (dismissed):
lgtm

harish1992 committed:
scripts: logging: dictionary: Add uart support

This commit adds support to dictionary logging to parse binary
data directly from input serial and generate the ascii logs.

Signed-off-by: Kumar Siddaramayya <harish.kumar@nordicsemi.no>
